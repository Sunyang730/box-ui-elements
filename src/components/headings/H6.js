// @flow
import * as React from 'react';
import classNames from 'classnames';

import './H6.scss';

type Props = {
    children: React.Node,
    className?: string,
};

const H6 = ({ children, className }: Props) => <h6 className={classNames('bdl-H6', className)}>{children}</h6>;

export default H6;
