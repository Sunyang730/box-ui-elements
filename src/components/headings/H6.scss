@import '../../styles/variables';

h6.bdl-H6 {
    @include font-family;

    color: $bdl-gray-62;
    font-size: 13px;
    font-weight: normal;
    letter-spacing: .3px;
    line-height: 20px;
}
