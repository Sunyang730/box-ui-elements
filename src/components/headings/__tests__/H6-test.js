// @flow
import * as React from 'react';

import H6 from '../H6';

describe('components/headings/H6', () => {
    const getWrapper = (props = {}) => shallow(<H6 {...props}>foo</H6>);

    test('should render H6', () => {
        const wrapper = getWrapper({ className: 'moo' });

        expect(wrapper).toMatchSnapshot();
    });
});
