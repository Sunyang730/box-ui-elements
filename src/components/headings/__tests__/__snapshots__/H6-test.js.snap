// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/headings/H6 should render H6 1`] = `
<h6
  className="bdl-H6 moo"
>
  foo
</h6>
`;
